### ISSUE TITLE
Node.js ES feature support table / visualization

### ISSUE THREAD

**Issue by PatrickHeneise** [openjs-foundation/cross-project-council#1072]
During the collaboration summit, there was a point to have a feature comparison table that outlines what ES/ECMA features are available in which Node.js version.

On Slack this discussion was supported by @AugustinMauroy:

> I propose to have an image/svg to see new feat are waiting (urlPattern). Also to see what are the Ecma feats are implement or not
> which feature we will not adopt like navigator object

This reminded me of https://node.green which has been out there for years and is still somewhat popular amongst developers. It's updated with the latest Node and ECMA features, but could use visual enhancements and lacks accessibility.

I reached out to William who's running the website and he'd be open to transfer the project and the domain to the foundation.

Opening the conversation here

**Comment by ljharb** [comment]
Indeed, node.green is that table. I'm already a maintainer on that repo, and would support moving it into the foundation if @williamkapke is amenable.

**Comment by williamkapke** [comment]
Yes. I still think it should belong with the foundation. 
When/if ready and approved, please reach out and I will complete the domain/repo/whatever transfers.

Thanks @PatrickHeneise for opening this issue.

**Comment by ljharb** [comment]
@williamkapke we'll need to file an application - per https://github.com/openjs-foundation/cross-project-council/blob/HEAD/PROJECT_PROGRESSION.md#project-proposal-requirements. I'll be happy to help with as much as possible, but I think you may need to do a bit of the work.

**Comment by PatrickHeneise** [comment]
@williamkapke did you find some time to file the application? Otherwise I'll close this as not planned. Thanks!

**Comment by williamkapke** [comment]
Hey Patrick!

No I will not be submitting an application. The request is absurd.
If the foundation doesn't want me to GIVE them the domain (and transfer the repo?) then so be it.
I'll just sell the domain.

It's will still be served via Github pages at https://williamkapke.github.io/node-compat-table as it is now with without the redirect to the domain.

**Comment by ljharb** [comment]
@williamkapke i'm happy to fill out the application, but there's legal reasons the project can't be just summarily given.

I would be happy to take the domain and the project and do the application process myself, if you prefer not to be involved?

**Comment by williamkapke** [comment]
Ok. Contact me william at kap dot co (not com) and lets get the domain transferred.
